lua/caching_handlers.lua

```diff
@@ -140,6 +141,7 @@ end
 -- fit the caching criteria OR because of some failures
 -- @cache_status: The value of the header Spectre-Cache-Status
 -- @incoming_zipkin_headers: Headers sent in from the request
+-- @original_status: The value of the header X-Original-Status
```

Comment:
seems this parameter is not passed (the comment refers to function params)

Reply:
ahh yes, will remove